src/Organic/Organic.php

```diff
@@ -14,7 +13,9 @@
 const CAMPAIGN_ASSET_META_KEY = 'empire_campaign_asset_guid';
 const GAM_ID_META_KEY = 'empire_gam_id';
 const SYNC_META_KEY = 'empire_sync';
-
+// The DSN to use before we load client-specific DSNs.
+const DEFAULT_SENTRY_DSN = 'https://e1cf660e5b3947a4bdf7c516afaaa7d2@o472819.ingest.sentry.io/4505048050434048';
```

Comment:
Sentry says it's "OK" for DSN keys to be public. (See https://docs.sentry.io/product/sentry-basics/dsn-explainer/.) This seems questionable to me. I'll deactivate this default and create a new one. We should pass in the default as a secret upon build.